src/plugins/telemetry_management_section/public/components/opt_in_security_example_flyout.tsx

```diff
@@ -0,0 +1,235 @@
+          />
+        </EuiCallOut>
+      );
+    }
+
+    return (
+      <EuiCodeBlock language="js">
+        {JSON.stringify(this.exampleSecurityPayload, null, 2)}
+      </EuiCodeBlock>
+    );
+  }
+
+  render() {
+    return (
+      <EuiPortal>
+        <EuiFlyout ownFocus onClose={this.props.onClose} maxWidth={true}>
+          <EuiFlyoutHeader>
+            <EuiTitle>
+              <h2>Endpoint security data</h2>
+            </EuiTitle>
+            <EuiTextColor color="subdued">
+              <EuiText>
+                This is a representative sample of the endpoint security alert event that we
+                collect. Endpoint security data is collected only when the Elastic Endpoint is
+                enabled. It includes information about the endpoint configuration and detection
+                events.
+              </EuiText>
+            </EuiTextColor>
+          </EuiFlyoutHeader>
+          <EuiFlyoutBody>{this.renderBody(this.state)}</EuiFlyoutBody>
+        </EuiFlyout>
+      </EuiPortal>
+    );
+  }
+
+  exampleSecurityPayload = {
+    '@timestamp': '2020-09-22T14:34:56.82202300Z',
+    agent: {
+      build: {
+        original:
+          'version: 7.9.1, compiled: Thu Aug 27 14:50:21 2020, branch: 7.9, commit: b594beb958817dee9b9d908191ed766d483df3ea',
+      },
+      id: '22dd8544-bcac-46cb-b970-5e681bb99e0b',
+      type: 'endpoint',
+      version: '7.9.1',
+    },
+    Endpoint: {
+      policy: {
+        applied: {
+          artifacts: {
+            global: {
+              identifiers: [
+                {
+                  sha256: '6a546aade5563d3e8dffc1fe2d93d33edda8f9ca3e17ac3cc9ac707620cb9ecd',
+                  name: 'endpointpe-v4-blocklist',
+                },
+                {
+                  sha256: '04f9f87accc5d5aea433427bd1bd4ec6908f8528c78ceed26f70df7875a99385',
+                  name: 'endpointpe-v4-exceptionlist',
+                },
+                {
+                  sha256: '1471838597fcd79a54ea4a3ec9a9beee1a86feaedab6c98e61102559ced822a8',
+                  name: 'endpointpe-v4-model',
+                },
+                {
+                  sha256: '824859b0c6749cc31951d92a73bbdddfcfe9f38abfe432087934d4dab9766ce8',
+                  name: 'global-exceptionlist-windows',
+                },
+              ],
+              version: '1.0.0',
+            },
+            user: {
+              identifiers: [
+                {
+                  sha256: 'd801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658',
+                  name: 'endpoint-exceptionlist-windows-v1',
+                },
+              ],
+              version: '1.0.0',
+            },
+          },
+        },
+      },
+    },
+    ecs: {
+      version: '1.5.0',
+    },
+    elastic: {
+      agent: {
+        id: 'b2e88aea-2671-402a-828a-957526bac315',
+      },
+    },
+    file: {
+      path: 'C:\\Users\\User\\Downloads\\mimikatz_trunk\\x64\\mimikatz.exe',
```

Comment:
Let's make this `path` set to `C:\\Windows\\Temp\\mimikatz.exe` which is a more realistic example path, and change the `directory` below to match as well. Otherwise, agreed this is a good representative sample alert.